docs/src/extend/plugins.md

```diff
@@ -78,7 +82,9 @@ module.exports = {
 
 ### Configs in Plugins
 
-You can bundle configurations inside a plugin by specifying them under the `configs` key. This can be useful when you want to provide not just code style, but also some custom rules to support it. Multiple configurations are supported per plugin. Note that it is not possible to specify a default configuration for a given plugin and that users must specify in their configuration file when they want to use one.
+You can bundle configurations inside a plugin by specifying them under the `configs` key. This can be useful when you want to bundle a set of custom rules with additional configuration. Multiple configurations are supported per plugin.
+
+It is not possible to specify a default configuration for a given plugin. Users must specify a default configuration in their configuration file.
```

Comment:
"Users _must_ specify a _default_ configuration in their configuration file." sounds confusing.

* "must" - sounds like users are required to use a configuration from the plugin, but they're not.
* "default" - if the user wants to use a configuration from the plugin, they specify it in their configuration file so there's no default.